src/cli/assert_node_engine_version_spec.ts

```diff
@@ -1,18 +1,33 @@
 import assert from 'assert'
-import { assertNodeEngineVersion } from './assert_node_engine_version'
+import { expect } from 'chai'
+import { validateNodeEngineVersion } from './assert_node_engine_version'
 
-describe(assertNodeEngineVersion.name, () => {
-  it('fails when the version is lower than specified in package.json', () => {
-    assert.throws(() =>
-      assertNodeEngineVersion('v11.0.0', () => ({
-        engines: {
-          node: '>=12',
-        },
-      }))
-    )
+describe(validateNodeEngineVersion.name, () => {
+  it('returns an error message when the version is lower than specified in package.json', () => {
+    // Arrange
+
+    // Act
+    const error = validateNodeEngineVersion('v11.0.0', () => ({
+      engines: {
+        node: '>=12',
+      },
+    }))
+
+    // Assert
+    expect(error).to.eql("Cucumber can only run on Node.js versions >=12. This Node.js version is 11.0.0")
   })
 
-  it('passes when the version is greater than specified in package.json', () => {
-    assertNodeEngineVersion('v17.0.0')
+  it('returns null when the version is greater than specified in package.json', () => {
+    // Arrange
+
+    // Act
+    const result = validateNodeEngineVersion('v17.0.0', () => ({
+      engines: {
+        node: '>=12',
+      },
+    }))
```

Comment:
Think its better to hardcode the engines here as well so this doesn't suddenly fail when we no longer support version 17 (probably a little ways off)

Reply:
💯, I missed that.

Reply:
Oh true, the list right now is 12, 14, 16, 17